src/UserService/Startup.cs

```diff
@@ -224,11 +231,20 @@ public void ConfigureServices(IServiceCollection services)
 
             ConfigureMassTransit(services);
 
+            //this will be used when all validation takes place on the pipeline
```

Comment:
Add word TODO, that all can see issues in IDE

Reply:
ok